### PR TITLE
OnboardingAPI: Implement faucet for remark signing evm on cent chain

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

When onboarding with evm on cent chain, if your DEVEL balance is too low we now pay for the remark signing tx.

Closes #<issue-number>

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/pages/Onboarding/queries/useSignRemark.ts [hunk 1]
```diff
@@ -113,23 +116,31 @@ export const useSignRemark = (
 
   const signSubstrateRemark = async (args: [message: string]) => {
     setIsSubstrateTxLoading(true)
-    if (!isEvmOnSubstrate && balances?.native.balance?.toDecimal().lt(expectedTxFee.mul(1.1))) {
+    if (balances?.native.balance?.toDecimal().lt(expectedTxFee.mul(1.1))) {
+      await getBalanceForSigning()
+    }
+    if (isEvmOnSubstrate && evmBalance?.toDecimal().lt(expectedTxFee.mul(1.1))) {
```
Comment (onnovisser) [inline centrifuge-app/src/pages/Onboarding/queries/useSignRemark.ts:122]: why is this checking `evmBalance` EVM on Substrate transactions use substrate balance

Comment (sophialittlejohn) [reply centrifuge-app/src/pages/Onboarding/queries/useSignRemark.ts:122]: it was the only way I could get the native (DEVEL) balance when connect to evm on cent chain. I'm considering evm on substrate to still be a substrate transaction

Comment (onnovisser) [reply centrifuge-app/src/pages/Onboarding/queries/useSignRemark.ts:122]: makes sense 👍

**Review by onnovisser — APPROVED** [review]
👍

**Pull request merged**